rust/Cargo.toml.in

```diff
@@ -41,11 +41,15 @@ aes-gcm = "~0.9.4"
 
 sawp-modbus = "~0.11.0"
 sawp = "~0.11.0"
-der-parser = "~6.0"
-kerberos-parser = "~0.7.1"
+der-parser = "~8.2.0"
+kerberos-parser = { version = "~0.7.1", default_features = false }
+
+# Kerberos parsing still depends on der-parser 6.0.1.
+der-parser6 = { package = "der-parser", version = "~6.0.1", default_features = false }
```

Comment:
@chifflier Would it make sense for for crates like `kerberos-parser` to re-export some of their dependencies so this can be less of a concern on the Suricata side?